dev/ledger.md

```diff
@@ -110,9 +110,9 @@ The block header contains the following components:
 
  - The block's _expired participation accounts_, which contains an optional slice of
    public keys of accounts. These accounts are expected to have their participation
-   key expire by the end of the round. The msgpack representation of the components are
+   key expire by the end of the round (or was expired before the current round). The msgpack representation of the components are
    described in detail below.  The slice is stored in msgpack key `partupdrmv`.
-
+The block's _expired participation accounts_ slice is valid as long as the participation keys of all the accounts in the slice are expired by the end of the round or were expired before, and the accounts themselves would have been online at the end of the round if they were not included in the slice. A block proposer may not include all such accounts in the slice and may even omit the slice completely.
```

Comment:
Could you replace the last sentence with the following, for better clarity ?
```
A block proposer can propose to reset a list of accounts to become offline accounts. The said accounts need to be such that are both online, and having their participation key expired. The validators would validate that the address list is valid by testing the individual accounts specified on that list.
```

Reply:
@fabrice102 - do you still want to make the proposed change, or should we keep it as is ?